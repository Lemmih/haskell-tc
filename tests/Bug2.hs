module Bug2 where

list = (\x -> x) []
