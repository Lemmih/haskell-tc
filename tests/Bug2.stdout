Bindings:
:: [v2]
  list = (\[4mx[0m -> x) []

Proofs:
coercion: âˆ€ v2. [v2]
  [4mlist = (\x -> x) [][0m
coercion: (âˆ€ a. [a]) @ v2
  list = (\x -> x) [4m[][0m
coercion: [v2]
  list = (\x -> [4mx[0m) []
